common/lib/transport/connectionmanager.js

```diff
@@ -329,10 +368,13 @@ var ConnectionManager = (function() {
 	/**
 	 * Called when an upgrade transport is connected,
 	 * to schedule the activation of that transport.
-	 * @param transport, the transport instance
-	 * @param connectionKey
+	 * @param error
+	 * @param transport
+	 * @param connectionId
+	 * @param connectionDetails
+	 * @param connectedMessage
 	 */
-	ConnectionManager.prototype.scheduleTransportActivation = function(error, transport, connectionKey, preSyncConnectionSerial, connectionId, connectionDetails) {
+	ConnectionManager.prototype.scheduleTransportActivation = function(error, transport, connectionId, connectionDetails, upgradeConnectedMessage) {
```

Comment:
You've called an object with either a connectionSerial or a timeSerial property a `connectionPosition` elsewhere, why've you called it an `upgradeConnectedMessage` here?